metasrv/src/meta_service/raftmeta.rs

```diff
@@ -137,19 +174,15 @@ impl MetaNodeBuilder {
             running_rx: rx,
             join_handles: Mutex::new(Vec::new()),
             joined_tasks: AtomicI32::new(1),
+            endpoint: endpoint.clone(),
+            metrics: RwLock::new(None),
```

Comment:
`MetaNode.raft.metrics().borrow().clone()` will return a clone of the current metrics, if this is enough for your need.